docs/Makefile

```diff
@@ -228,3 +229,21 @@ dummy:
 	$(SPHINXBUILD) -b dummy $(ALLSPHINXOPTS) $(BUILDDIR)/dummy
 	@echo
 	@echo "Build finished. Dummy builder generates no files."
+
+.PHONY: collectstatic
+collectstatic:
+	npm install
```

Comment:
![screencapture-circleci-gh-novafloss-django-formidable-748-1488986984853](https://cloud.githubusercontent.com/assets/251655/23710284/7b2efc5a-041c-11e7-8ae5-8c37a0a1fdf9.png)

I tried to build the docs on CircleCI: worked pretty well!